src/spriteSheet.js

```diff
@@ -32,6 +32,7 @@
       this.spriteSheet = properties.spriteSheet;
       this.frames = properties.frames;
       this.frameRate = properties.frameRate;
+      this.loop = properties.loop || true;
```

Comment:
If `properties.loop` is `false`, this will default it to `true`. Should check that it's not equal to `undefined` else set it to `true`.